plugins/pipelines/syncPRs.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const idSchema = joi.reach(schema.models.pipeline.base, 'id');
+
+module.exports = () => ({
+    method: 'GET',
```

Comment:
This question is more about semantics than the logic. Is a `GET` operation appropriate for syncing the pull requests?